docs/technical-reference/running-modes.md

```diff
@@ -1,22 +1,15 @@
 # Running Modes
 
-## Deployment / Delivery models
+Lifecycle Manager (and module operators) can run in two modes:
 
-lifecycle-manager (and module operators) can run in 2 modes:
+- single-cluster - deployment mode in which Lifecycle Manager is running on the same clutser on which you deploy Kyma. This mode doesn't require Kyma or ModuleTemplate CRs [synchronization](../technical-reference/api/README.md#synchronization-of-module-catalog-with-remote-clusters).
+- control-plane - deployment mode in which Lifecycle Manager is running on the central Kubernetes cluster that manages multiple remote clusters that are targets for Kyma installations. In this mode, Kyma and ModuleTemplate CRs are synchronized between the central cluster and remote ones. Access to remote clusters is configured using centrally-managed K8s Secrets with `kubeconfig` files with client configurations.
```

Comment:
The Secrets do not contain files, they contain the same data as the kubeconfig file... We don't have to be very specific here. Perhaps this is enough:
```suggestion
- control-plane - deployment mode in which Lifecycle Manager is running on the central Kubernetes cluster that manages multiple remote clusters that are targets for Kyma installations. In this mode, Kyma and ModuleTemplate CRs are synchronized between the central cluster and remote ones. Access to remote clusters is enabled via centrally-managed K8s Secrets with required connection configuration.
```